mlir/test/Quantum/ScatterTest.mlir

```diff
@@ -155,28 +175,18 @@ func.func public @full_example_scatter(%input: tensor<3x4x2xi64>, %update: tenso
     // CHECK-NEXT:   [[RESTENSOR:%.+]] = tensor.from_elements [[RES]] : tensor<i64>
     // CHECK-NEXT:   return [[RESTENSOR]] : tensor<i64>
 
-// CHECK: [[CALL1:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL2:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL3:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL4:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL5:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL6:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL7:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL8:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL9:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL10:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL11:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL12:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL13:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL14:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL15:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL16:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL17:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL18:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL19:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL20:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL21:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL22:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL23:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK: [[CALL24:%.+]] = call @__catalyst_update_scatter[[NUMBER0]](
-// CHECK-NOT: call @__catalyst_update_scatter[[NUMBER0]](
\ No newline at end of file
+    // CHECK:    [[CST:%.+]] = arith.constant dense<[[DENSE:.*]]> : tensor<24x4xindex>
```

Comment:
There should be another check line to differentiate from the function scope above.